vip-integrations.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+use InvalidArgumentException;
+
+defined( 'ABSPATH' ) || die();
+
+require_once __DIR__ . '/vip-integration-helper/integrations/integration.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/integrations.php';
+require_once __DIR__ . '/vip-integration-helper/integrations/block-data-api.php';
+
+class VipIntegrations {
```

Comment:
What's the benefit is having this class separate from the `Integrations` class? Feels like they could be combined?

Reply:
The `Integrations` class is just a collection - it should know nothing about WordPress.

Reply:
^ Following some of @GaryJones previous advice, we did some more work to make most integrations code dumber and unaware of WordPress. This meant we were able to [remove slow test isolation](https://github.com/Automattic/vip-go-mu-plugins/pull/4458/commits/78ddb24506af230f1f1673fa9a7df3a975a21869#diff-64b428d68bfd6773d14920ee388f49ef7f4807037d8fb196be4ab8e1930d9961L9-L12) and generally made tests a lot cleaner.